app/services/post_scraper.rb

```diff
@@ -1,23 +1,23 @@
 class PostScraper < Object
   SANDBOX_ID = Rails.env.production? ? 3 : 5
   BASE_ACCOUNTS = {
-    'alicornucopia' => 'Alicorn',
-    'pythbox' => 'Kappa',
-    'lintamande' => 'lintamande',
-    'marrinikari' => 'Marri',
-    'peterxy' => 'Pedro',
-    'peterverse' => 'Pedro',
-    'curiousdiscoverer' => 'CuriousDiscoverer',
-    'aestrix' => 'Aestrix',
-    'unbitwise' => 'Unbitwise',
-    'erinflight' => 'ErinFlight',
-    'andaisq' => 'andaisq',
-    'rockeye-stonetoe' => 'Rockeye',
-    'rockeye_stonetoe' => 'Rockeye',
+    'alicornucopia'      => 'Alicorn',
+    'pythbox'            => 'Kappa',
+    'lintamande'         => 'lintamande',
+    'marrinikari'        => 'Marri',
+    'peterxy'            => 'Pedro',
+    'peterverse'         => 'Pedro',
+    'curiousdiscoverer'  => 'CuriousDiscoverer',
+    'aestrix'            => 'Aestrix',
+    'unbitwise'          => 'Unbitwise',
+    'erinflight'         => 'ErinFlight',
+    'andaisq'            => 'andaisq',
+    'rockeye-stonetoe'   => 'Rockeye',
+    'rockeye_stonetoe'   => 'Rockeye',
     'maggie-of-the-owls' => 'MaggieoftheOwls',
     'maggie_of_the_owls' => 'MaggieoftheOwls', # have both - and _ versions cause Dreamwidth supports both
-    'nemoconsequentiae' => 'Nemo',
-    'armokgob' => 'Armok',
+    'nemoconsequentiae'  => 'Nemo',
+    'armokgob'           => 'Armok',
```

Comment:
This style might make it a bit of a nightmare to maintain. Are we able to disable it for certain areas? Or maybe completely, actually, I'm not sure we care all that much about keeping `=>` aligned like this.

Reply:
it's _so much easier to parse_ tho, at least to me. (And the cop does auto-correction so I don't think it'd be hard to maintain, but I do see your point)

Reply:
Hmmm, okay. I don't mind too much.